Tests/GitHubProjects.tests.ps1

```diff
@@ -0,0 +1,365 @@
+            }
+
+            $null = New-GitHubProject -OwnerName $script:ownerName -RepositoryName $repo.name -Name $defaultRepoProject -Description $defaultRepoProjectDesc
+            $results = Get-GitHubProject -OwnerName $script:ownerName -RepositoryName $repo.name -Name $defaultRepoProject
+            It 'Project Exists' {
+                $results | Should Not BeNullOrEmpty
+            }
+
+            It 'Name is correct' {
+                $results.name | Should be $defaultRepoProject
+            }
+
+            It 'Description should be updated' {
+                $results.body | Should be $defaultRepoProjectDesc
+            }
+        }
+    }
+
+    Describe 'Remove Project' {
+        Context 'Remove User projects' {
+            BeforeAll {
+                $null = New-GitHubProject -UserProject -Name $defaultUserProject -Description $defaultUserProjectDesc
+            }
+
+            $null = Remove-GitHubProject -UserName $script:ownerName -Name $defaultUserProject
+            $results = Get-GitHubProject -UserName $script:ownerName -Name $defaultUserProject
+            It 'Project should be removed' {
+                $results | Should BeNullOrEmpty
+            }
+        }
+
+        Context 'Remove Organization projects' {
+            BeforeAll {
+                $null = New-GitHubProject -OrganizationName $script:organizationName -Name $defaultOrgProject -Description $defaultOrgProjectDesc
+            }
+
+            $null = Remove-GitHubProject -OrganizationName $script:organizationName -Name $defaultOrgProject
+            $results = Get-GitHubProject -OrganizationName $script:organizationName -Name $defaultOrgProject
+            It 'Project should be removed' {
+                $results | Should BeNullOrEmpty
+            }
+        }
+
+        Context 'Remove Repo projects' {
+            BeforeAll {
+                $null = New-GitHubProject -OwnerName $script:ownerName -RepositoryName $repo.name -Name $defaultRepoProject -Description $defaultRepoProjectDesc
+            }
+
+            $null = Remove-GitHubProject -OwnerName $script:ownerName -RepositoryName $repo.name -Name $defaultRepoProject
+            $results = Get-GitHubProject -OwnerName $script:ownerName -RepositoryName $repo.name -Name $defaultRepoProject
+            It 'Project should be removed' {
+                $results | Should  BeNullOrEmpty
+            }
+        }
+
+        Context 'Remove project by id' {
+            BeforeAll {
+                $repoProject = New-GitHubProject -OwnerName $script:ownerName -RepositoryName $repo.name -Name $defaultRepoProject -Description $defaultRepoProjectDesc
+            }
+
+            $null = Remove-GitHubProject -Project $repoProject.id
+
+            try
+            {
+                $results = Get-GitHubProject -Project $repoProject.id
+            }
+            catch {
+                $null
```

Comment:
What's the scenario you're working around here?  At the very least, this deserves a comment.

Reply:
The `Get` was failing to find the project and when using the `-Project`  parameter it threw an error which messed up the tests.

Changed these tests around to test for the thrown error instead... new version incoming :)